src/ports.ts

```diff
@@ -50,5 +50,11 @@ export function getExternalPort(app: WithConfig) {
   if (Number(process.env?.EXTERNALPORT) > 0) {
     return Number(process.env?.EXTERNALPORT)
   }
+  if (app.config.settings.proxy_port) {
+    return app.config.settings.proxy_port
+  }
+  if (app.config.port) {
```

Comment:
Huh, quite a mess with the different env variables and settings. We have both `app.config.port` (used here, independent of ssl being on or off) and  `app.config.settings.port` (used for non ssl port).

The ui uses `settings.port | sslport` in [PUT settings](https://github.com/SignalK/signalk-server/blob/aac941959b303ce9b0ec69eabbcfb1f97be0b11f/src/serverroutes.ts#L603-L609)

Maybe we should just remove `app.config.port`?

Reply:
If it simplifies things then I'm ok with it.